Makefile

```diff
@@ -15,7 +17,14 @@ vagrant-up:
 aws: aws-provision ansible-galaxy site
 
 aws-provision: 
-	ansible-playbook -i localhost, aws-provision.yml -v --ask-vault-pass
+	ansible-playbook -i localhost, aws-provision.yml -v
 
 site: 
-	ansible-playbook -i ec2.py site.yml -v --ask-vault-pass
+	ansible-playbook -i ec2.py site.yml -v
+
+import-gpg-keys:
+	$(foreach var,$(shell cat gpg.recipients | awk -F: '{print $$1}'),gpg --keyserver hkp://keyserver.ubuntu.com --recv-keys $(var);)
```

Comment:
Can we merge this one in and then do a follow up PR to do the refactoring of the Makefile? It seems a shame to miss the thread of how we got there.